src/components/TeaserCarousel/Row.js

```diff
@@ -59,10 +59,6 @@ const styles = {
     bottom: 0,
     opacity: 0.7
   }),
-  arrowIcon: css({
-    // ontop of arrowBg
-    position: 'relative'
-  }),
```

Comment:
removing this creates a visual regression:
<img width="533" alt="Screenshot 2021-04-19 at 16 07 13" src="https://user-images.githubusercontent.com/410211/115249889-5a8f3d80-a129-11eb-8bad-3fcf0b8b85ee.png">

Reply:
done